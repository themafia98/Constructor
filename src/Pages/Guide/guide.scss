
.Guide{

    .guide--flex{
        display: flex;
        flex-direction: column;
    }
    .Guide__about{
        text-align: center;
        font-size: 1.3rem;
        p:not(.Guide__about__rev) {
            padding: 0;
            margin: 0;
        }
        h2 {
            margin-bottom: 10px;
        }

        .Guide__about__rev {
            font-weight: bold;
        }
    }

    .guide {
        display: flex;
        flex-direction: column;
        align-items: center;
        padding-left: 2px;

        .guide__title {
            margin: 0;
            margin-top: 15px;
            font-size: 1.3rem;
        }
        .guide_content{
            font-size: 1rem;
            font-weight: bold;
        }
    }
}