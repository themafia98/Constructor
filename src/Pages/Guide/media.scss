@media screen and (max-width: 600px){
    .Guide{
        .Guide__about {
            text-align: center;
            font-size: 1rem;
        }

        .guide_content {
            text-align: center;
            width: 50%;
            margin: 0;
            margin-bottom: 5px;
        }

        .guide{
            [alt='guide'] {
                width: 95%;
                height: 50%;
            }
        }
    }
}