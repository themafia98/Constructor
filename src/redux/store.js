import {createStore} from 'redux';
import combineReducers from './reducers';

let broswerPluginDev = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__();
const store = createStore(combineReducers,broswerPluginDev);
export default store;