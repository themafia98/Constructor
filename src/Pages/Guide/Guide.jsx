import React,{Fragment} from 'react';
import PropTypes from 'prop-types';
import {Redirect} from 'react-router-dom';
import {connect} from 'react-redux';

import SwitchContainer from '../../components/SwitchContainer/SwitchContainer';
import withFirebase from '../../components/withFirebase';
import Header from '../../components/header/Header';
import Loader from '../../components/loading/Loader';

import './guide.scss';
class Guide extends React.PureComponent {

    static propTypes = {
        config: PropTypes.object.isRequired, /** @Config app */
        firebase: PropTypes.object.isRequired, /** @firebase class for use firebase functions */
        history: PropTypes.object, /** @Router HTML5 history */
        location: PropTypes.object, /** @Router */
        match: PropTypes.object, /** @Router */
    }

    state = {
        config: this.props.config
    };

    render(){

        let user = this.props.firebase.getCurrentUser();
        if (user && this.props.idUser) {
            const { config } = this.state;
            return (
                <Fragment>
                    <Header title = {config.title} idUser = {user.uid} />
                    <section className = 'Guide'>
                        <div className = 'container guide--flex'>
                            <div className = 'Guide__about'>
                                <h2>Documentation</h2>
                                <p>Get started with {config.title}</p>
                                <p className = 'Guide__about__rev'>
                                    The Constructor guides are step-by-step walkthroughs
                                    that help you get started using Constructor.
                                </p>
                            </div>
                            <SwitchContainer />
                        </div>
                    </section>
                </Fragment>
            )
        } else if (!user) return <Redirect to = { '/'} />
        else return <Loader path = '/img/loading.gif' type = 'About' />

    }
}

const mapStateToProps = (state) => {
    return {
      idUser: state.cabinet.idUser,
    }
  };

export default connect(mapStateToProps)(withFirebase(Guide));