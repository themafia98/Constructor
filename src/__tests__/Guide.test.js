import React from 'react';
import firebase from 'firebase';
import {Provider} from 'react-redux';
import {MemoryRouter, Route} from 'react-router-dom';
import renderer from 'react-test-renderer';
import Guide from '../Pages/Guide/Guide';
import store from '../redux/testReduxStore';



const config = require('../config.json');

const GuideComponent = renderer.create(
    <MemoryRouter initialEntries = {['/Cabinet/Guide']}>
    <Route path = "/Cabinet/Guide" exact >
        <Provider store = {store}>
            <Guide config = {config} firebase = {firebase} />
        </Provider>
    </Route>
    </MemoryRouter>
);

test("Guide test", function(){

    let snapshotGuideComponent = GuideComponent.toJSON();
    expect(snapshotGuideComponent).toMatchSnapshot();

    const panel = GuideComponent.root.find(item => item.props.className === 'switch__button panel activeButton');
    panel.props.onClick({stopPropagation: () => null});

    snapshotGuideComponent = GuideComponent.toJSON();
    expect(snapshotGuideComponent).toMatchSnapshot();

    const add = GuideComponent.root.find(item => item.props.className === 'switch__button add');
    add.props.onClick({stopPropagation: () => null});

    snapshotGuideComponent = GuideComponent.toJSON();
    expect(snapshotGuideComponent).toMatchSnapshot();

    const section = GuideComponent.root.find(item => item.props.className === 'switch__button section');
    section.props.onClick({stopPropagation: () => null});

    snapshotGuideComponent = GuideComponent.toJSON();
    expect(snapshotGuideComponent).toMatchSnapshot();

    const project = GuideComponent.root.find(item => item.props.className === 'switch__button project');
    project.props.onClick({stopPropagation: () => null});

    snapshotGuideComponent = GuideComponent.toJSON();
    expect(snapshotGuideComponent).toMatchSnapshot();
});