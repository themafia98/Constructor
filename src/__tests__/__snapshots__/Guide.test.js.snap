// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guide test 1`] = `
Array [
  <header>
    <div
      className="container"
    >
      <div
        className="flex-row"
      >
        <div
          className="header__CabinetInfo"
          onClick={[Function]}
        >
          <img
            alt="icon"
            className="mainIcon"
            draggable={false}
            onClick={null}
            src="/img/logo.svg"
            title={null}
          />
          <h3
            className="tilteApp"
          >
            Constructor
          </h3>
        </div>
        <div
          className="Navigator"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-w-14  about fas fa-2x fa-book"
            data-icon="book"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M448 360V24c0-13.3-10.7-24-24-24H96C43 0 0 43 0 96v320c0 53 43 96 96 96h328c13.3 0 24-10.7 24-24v-16c0-7.5-3.5-14.3-8.9-18.7-4.2-15.4-4.2-59.3 0-74.7 5.4-4.3 8.9-11.1 8.9-18.6zM128 134c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm0 64c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm253.4 250H96c-17.7 0-32-14.3-32-32 0-17.6 14.4-32 32-32h285.4c-1.9 17.1-1.9 46.9 0 64z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <img
            alt="icon"
            className="about"
            draggable={false}
            onClick={[Function]}
            src="/img/about_logo.svg"
            title="about"
          />
          <img
            alt="icon"
            className="logoutButton"
            draggable={false}
            onClick={[Function]}
            src="/img/logOut.svg"
            title="logOut"
          />
        </div>
      </div>
    </div>
  </header>,
  <section
    className="Guide"
  >
    <div
      className="container guide--flex"
    >
      <div
        className="Guide__about"
      >
        <h2>
          Documentation
        </h2>
        <p>
          Get started with 
          Constructor
        </p>
        <p
          className="Guide__about__rev"
        >
          The Constructor guides are step-by-step walkthroughs that help you get started using Constructor.
        </p>
      </div>
      <div
        className="switchContainer"
      >
        <div
          className="col--17 col-sm-10 switchContainer__controllers"
        >
          <img
            alt="icon"
            className="switchContainer__menu"
            draggable={false}
            onClick={null}
            src="/img/Menu_Burder.png"
            title={null}
          />
          <ul
            className="guide_list list"
          >
            <li>
              <input
                className="switch__button panel activeButton"
                onClick={[Function]}
                type="button"
                value="Open panel for edit"
              />
            </li>
            <li>
              <input
                className="switch__button search"
                onClick={[Function]}
                type="button"
                value="Search image/media"
              />
            </li>
            <li>
              <input
                className="switch__button add"
                onClick={[Function]}
                type="button"
                value="Adding components"
              />
            </li>
            <li>
              <input
                className="switch__button section"
                onClick={[Function]}
                type="button"
                value="Adding new section"
              />
            </li>
            <li>
              <input
                className="switch__button project"
                onClick={[Function]}
                type="button"
                value="Create new project"
              />
            </li>
          </ul>
        </div>
        <div
          className="col--80--1"
        >
          <div
            className="guide openPanelGuide"
          >
            <h4
              className="guide__title"
            >
              Open panel
            </h4>
            <p
              className="guide_content"
            >
              If you want to open the toolbar, you should double-click on the background.
            </p>
            <img
              alt="guide"
              src="/img/guideOpenPanel.gif"
            />
          </div>
        </div>
      </div>
    </div>
  </section>,
]
`;

exports[`Guide test 2`] = `
Array [
  <header>
    <div
      className="container"
    >
      <div
        className="flex-row"
      >
        <div
          className="header__CabinetInfo"
          onClick={[Function]}
        >
          <img
            alt="icon"
            className="mainIcon"
            draggable={false}
            onClick={null}
            src="/img/logo.svg"
            title={null}
          />
          <h3
            className="tilteApp"
          >
            Constructor
          </h3>
        </div>
        <div
          className="Navigator"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-w-14  about fas fa-2x fa-book"
            data-icon="book"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M448 360V24c0-13.3-10.7-24-24-24H96C43 0 0 43 0 96v320c0 53 43 96 96 96h328c13.3 0 24-10.7 24-24v-16c0-7.5-3.5-14.3-8.9-18.7-4.2-15.4-4.2-59.3 0-74.7 5.4-4.3 8.9-11.1 8.9-18.6zM128 134c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm0 64c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm253.4 250H96c-17.7 0-32-14.3-32-32 0-17.6 14.4-32 32-32h285.4c-1.9 17.1-1.9 46.9 0 64z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <img
            alt="icon"
            className="about"
            draggable={false}
            onClick={[Function]}
            src="/img/about_logo.svg"
            title="about"
          />
          <img
            alt="icon"
            className="logoutButton"
            draggable={false}
            onClick={[Function]}
            src="/img/logOut.svg"
            title="logOut"
          />
        </div>
      </div>
    </div>
  </header>,
  <section
    className="Guide"
  >
    <div
      className="container guide--flex"
    >
      <div
        className="Guide__about"
      >
        <h2>
          Documentation
        </h2>
        <p>
          Get started with 
          Constructor
        </p>
        <p
          className="Guide__about__rev"
        >
          The Constructor guides are step-by-step walkthroughs that help you get started using Constructor.
        </p>
      </div>
      <div
        className="switchContainer"
      >
        <div
          className="col--17 col-sm-10 switchContainer__controllers"
        >
          <img
            alt="icon"
            className="switchContainer__menu"
            draggable={false}
            onClick={null}
            src="/img/Menu_Burder.png"
            title={null}
          />
          <ul
            className="guide_list list"
          >
            <li>
              <input
                className="switch__button panel activeButton"
                onClick={[Function]}
                type="button"
                value="Open panel for edit"
              />
            </li>
            <li>
              <input
                className="switch__button search"
                onClick={[Function]}
                type="button"
                value="Search image/media"
              />
            </li>
            <li>
              <input
                className="switch__button add"
                onClick={[Function]}
                type="button"
                value="Adding components"
              />
            </li>
            <li>
              <input
                className="switch__button section"
                onClick={[Function]}
                type="button"
                value="Adding new section"
              />
            </li>
            <li>
              <input
                className="switch__button project"
                onClick={[Function]}
                type="button"
                value="Create new project"
              />
            </li>
          </ul>
        </div>
        <div
          className="col--80--1"
        >
          <div
            className="guide openPanelGuide"
          >
            <h4
              className="guide__title"
            >
              Open panel
            </h4>
            <p
              className="guide_content"
            >
              If you want to open the toolbar, you should double-click on the background.
            </p>
            <img
              alt="guide"
              src="/img/guideOpenPanel.gif"
            />
          </div>
        </div>
      </div>
    </div>
  </section>,
]
`;

exports[`Guide test 3`] = `
Array [
  <header>
    <div
      className="container"
    >
      <div
        className="flex-row"
      >
        <div
          className="header__CabinetInfo"
          onClick={[Function]}
        >
          <img
            alt="icon"
            className="mainIcon"
            draggable={false}
            onClick={null}
            src="/img/logo.svg"
            title={null}
          />
          <h3
            className="tilteApp"
          >
            Constructor
          </h3>
        </div>
        <div
          className="Navigator"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-w-14  about fas fa-2x fa-book"
            data-icon="book"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M448 360V24c0-13.3-10.7-24-24-24H96C43 0 0 43 0 96v320c0 53 43 96 96 96h328c13.3 0 24-10.7 24-24v-16c0-7.5-3.5-14.3-8.9-18.7-4.2-15.4-4.2-59.3 0-74.7 5.4-4.3 8.9-11.1 8.9-18.6zM128 134c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm0 64c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm253.4 250H96c-17.7 0-32-14.3-32-32 0-17.6 14.4-32 32-32h285.4c-1.9 17.1-1.9 46.9 0 64z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <img
            alt="icon"
            className="about"
            draggable={false}
            onClick={[Function]}
            src="/img/about_logo.svg"
            title="about"
          />
          <img
            alt="icon"
            className="logoutButton"
            draggable={false}
            onClick={[Function]}
            src="/img/logOut.svg"
            title="logOut"
          />
        </div>
      </div>
    </div>
  </header>,
  <section
    className="Guide"
  >
    <div
      className="container guide--flex"
    >
      <div
        className="Guide__about"
      >
        <h2>
          Documentation
        </h2>
        <p>
          Get started with 
          Constructor
        </p>
        <p
          className="Guide__about__rev"
        >
          The Constructor guides are step-by-step walkthroughs that help you get started using Constructor.
        </p>
      </div>
      <div
        className="switchContainer"
      >
        <div
          className="col--17 col-sm-10 switchContainer__controllers"
        >
          <img
            alt="icon"
            className="switchContainer__menu"
            draggable={false}
            onClick={null}
            src="/img/Menu_Burder.png"
            title={null}
          />
          <ul
            className="guide_list list"
          >
            <li>
              <input
                className="switch__button panel activeButton"
                onClick={[Function]}
                type="button"
                value="Open panel for edit"
              />
            </li>
            <li>
              <input
                className="switch__button search"
                onClick={[Function]}
                type="button"
                value="Search image/media"
              />
            </li>
            <li>
              <input
                className="switch__button add"
                onClick={[Function]}
                type="button"
                value="Adding components"
              />
            </li>
            <li>
              <input
                className="switch__button section"
                onClick={[Function]}
                type="button"
                value="Adding new section"
              />
            </li>
            <li>
              <input
                className="switch__button project"
                onClick={[Function]}
                type="button"
                value="Create new project"
              />
            </li>
          </ul>
        </div>
        <div
          className="col--80--1"
        >
          <div
            className="guide openPanelGuide"
          >
            <h4
              className="guide__title"
            >
              Open panel
            </h4>
            <p
              className="guide_content"
            >
              If you want to open the toolbar, you should double-click on the background.
            </p>
            <img
              alt="guide"
              src="/img/guideOpenPanel.gif"
            />
          </div>
        </div>
      </div>
    </div>
  </section>,
]
`;

exports[`Guide test 4`] = `
Array [
  <header>
    <div
      className="container"
    >
      <div
        className="flex-row"
      >
        <div
          className="header__CabinetInfo"
          onClick={[Function]}
        >
          <img
            alt="icon"
            className="mainIcon"
            draggable={false}
            onClick={null}
            src="/img/logo.svg"
            title={null}
          />
          <h3
            className="tilteApp"
          >
            Constructor
          </h3>
        </div>
        <div
          className="Navigator"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-w-14  about fas fa-2x fa-book"
            data-icon="book"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M448 360V24c0-13.3-10.7-24-24-24H96C43 0 0 43 0 96v320c0 53 43 96 96 96h328c13.3 0 24-10.7 24-24v-16c0-7.5-3.5-14.3-8.9-18.7-4.2-15.4-4.2-59.3 0-74.7 5.4-4.3 8.9-11.1 8.9-18.6zM128 134c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm0 64c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm253.4 250H96c-17.7 0-32-14.3-32-32 0-17.6 14.4-32 32-32h285.4c-1.9 17.1-1.9 46.9 0 64z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <img
            alt="icon"
            className="about"
            draggable={false}
            onClick={[Function]}
            src="/img/about_logo.svg"
            title="about"
          />
          <img
            alt="icon"
            className="logoutButton"
            draggable={false}
            onClick={[Function]}
            src="/img/logOut.svg"
            title="logOut"
          />
        </div>
      </div>
    </div>
  </header>,
  <section
    className="Guide"
  >
    <div
      className="container guide--flex"
    >
      <div
        className="Guide__about"
      >
        <h2>
          Documentation
        </h2>
        <p>
          Get started with 
          Constructor
        </p>
        <p
          className="Guide__about__rev"
        >
          The Constructor guides are step-by-step walkthroughs that help you get started using Constructor.
        </p>
      </div>
      <div
        className="switchContainer"
      >
        <div
          className="col--17 col-sm-10 switchContainer__controllers"
        >
          <img
            alt="icon"
            className="switchContainer__menu"
            draggable={false}
            onClick={null}
            src="/img/Menu_Burder.png"
            title={null}
          />
          <ul
            className="guide_list list"
          >
            <li>
              <input
                className="switch__button panel activeButton"
                onClick={[Function]}
                type="button"
                value="Open panel for edit"
              />
            </li>
            <li>
              <input
                className="switch__button search"
                onClick={[Function]}
                type="button"
                value="Search image/media"
              />
            </li>
            <li>
              <input
                className="switch__button add"
                onClick={[Function]}
                type="button"
                value="Adding components"
              />
            </li>
            <li>
              <input
                className="switch__button section"
                onClick={[Function]}
                type="button"
                value="Adding new section"
              />
            </li>
            <li>
              <input
                className="switch__button project"
                onClick={[Function]}
                type="button"
                value="Create new project"
              />
            </li>
          </ul>
        </div>
        <div
          className="col--80--1"
        >
          <div
            className="guide openPanelGuide"
          >
            <h4
              className="guide__title"
            >
              Open panel
            </h4>
            <p
              className="guide_content"
            >
              If you want to open the toolbar, you should double-click on the background.
            </p>
            <img
              alt="guide"
              src="/img/guideOpenPanel.gif"
            />
          </div>
        </div>
      </div>
    </div>
  </section>,
]
`;

exports[`Guide test 5`] = `
Array [
  <header>
    <div
      className="container"
    >
      <div
        className="flex-row"
      >
        <div
          className="header__CabinetInfo"
          onClick={[Function]}
        >
          <img
            alt="icon"
            className="mainIcon"
            draggable={false}
            onClick={null}
            src="/img/logo.svg"
            title={null}
          />
          <h3
            className="tilteApp"
          >
            Constructor
          </h3>
        </div>
        <div
          className="Navigator"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-w-14  about fas fa-2x fa-book"
            data-icon="book"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M448 360V24c0-13.3-10.7-24-24-24H96C43 0 0 43 0 96v320c0 53 43 96 96 96h328c13.3 0 24-10.7 24-24v-16c0-7.5-3.5-14.3-8.9-18.7-4.2-15.4-4.2-59.3 0-74.7 5.4-4.3 8.9-11.1 8.9-18.6zM128 134c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm0 64c0-3.3 2.7-6 6-6h212c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H134c-3.3 0-6-2.7-6-6v-20zm253.4 250H96c-17.7 0-32-14.3-32-32 0-17.6 14.4-32 32-32h285.4c-1.9 17.1-1.9 46.9 0 64z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          <img
            alt="icon"
            className="about"
            draggable={false}
            onClick={[Function]}
            src="/img/about_logo.svg"
            title="about"
          />
          <img
            alt="icon"
            className="logoutButton"
            draggable={false}
            onClick={[Function]}
            src="/img/logOut.svg"
            title="logOut"
          />
        </div>
      </div>
    </div>
  </header>,
  <section
    className="Guide"
  >
    <div
      className="container guide--flex"
    >
      <div
        className="Guide__about"
      >
        <h2>
          Documentation
        </h2>
        <p>
          Get started with 
          Constructor
        </p>
        <p
          className="Guide__about__rev"
        >
          The Constructor guides are step-by-step walkthroughs that help you get started using Constructor.
        </p>
      </div>
      <div
        className="switchContainer"
      >
        <div
          className="col--17 col-sm-10 switchContainer__controllers"
        >
          <img
            alt="icon"
            className="switchContainer__menu"
            draggable={false}
            onClick={null}
            src="/img/Menu_Burder.png"
            title={null}
          />
          <ul
            className="guide_list list"
          >
            <li>
              <input
                className="switch__button panel activeButton"
                onClick={[Function]}
                type="button"
                value="Open panel for edit"
              />
            </li>
            <li>
              <input
                className="switch__button search"
                onClick={[Function]}
                type="button"
                value="Search image/media"
              />
            </li>
            <li>
              <input
                className="switch__button add"
                onClick={[Function]}
                type="button"
                value="Adding components"
              />
            </li>
            <li>
              <input
                className="switch__button section"
                onClick={[Function]}
                type="button"
                value="Adding new section"
              />
            </li>
            <li>
              <input
                className="switch__button project"
                onClick={[Function]}
                type="button"
                value="Create new project"
              />
            </li>
          </ul>
        </div>
        <div
          className="col--80--1"
        >
          <div
            className="guide openPanelGuide"
          >
            <h4
              className="guide__title"
            >
              Open panel
            </h4>
            <p
              className="guide_content"
            >
              If you want to open the toolbar, you should double-click on the background.
            </p>
            <img
              alt="guide"
              src="/img/guideOpenPanel.gif"
            />
          </div>
        </div>
      </div>
    </div>
  </section>,
]
`;
